Sources/TensorFlow/Core/LazyTensorTFFunctionBuilder.swift

```diff
@@ -0,0 +1,315 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import CTensorFlow
+
+class TFGraph {
```

Comment:
Could you add a doc comment with a summary of what this is for?

Reply:
Done.

Reply:
"also exposes the some" -> "also exposes some"

Reply:
Also, why do you refer to it as `TF_Graph*` and `TF_Function*` instead of `TF_Graph` and `TF_Function`, respectively?

Reply:
Changed them to `TF_Graph` and `TF_Function`.